composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "findologic/plugin-shopware-6",
   "description": "FINDOLOGIC plugin for Shopware 6 e-commerce system",
-  "version": "v2.1.2",
+  "version": "2.2.0",
```

Comment:
Version was with `v` before.

Reply:
Yes the FroshPluginUploader does not allow versions with prefix `v`, so we changed this.